salt/addons/dex/manifests/15-configmap.yaml

```diff
@@ -43,6 +43,38 @@ data:
           groupAttr: uniqueMember
 
           nameAttr: cn
+    {% for con in salt['pillar.get']('dex:connectors', []) %}
+    {% if con['type'] == 'ldap' %}
+    - type: ldap
+      id: "{{ con['id'] }}"
+      name: "{{ con['name'] }}"
+      config:
+        host: "{{ con['server'] }}"
+        startTLS: {{ con['start_tls'] }}
+        {% if con['bind']['anonymous'] %}
+        {% else %}
```

Comment:
Couldn't we replace this

```jinja
        {% if con['bind']['anonymous'] %}
        {% else %}
        ...
```

by something like

```jinja
        {% if not con['bind']['anonymous'] %}
        ...
```

?

Reply:
I agree that it would be equivalent and be one less line. Now that you point it out though it would really be best I think to output a comment into the configuration file instead of nothing when anonymous is used. That would make it clear that the lack of the bind information is telling dex to do an anonymous bind, which is unclear looking at the configuration file without a comment being present.

Reply:
Yeah, I think you should at least add a comment clarifying why you are leaving an empty `if` statement...